fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/VersionIndexTest.java

```diff
@@ -753,39 +871,39 @@ public void updateWithinContext(int testFormatVersion, boolean testSplitLongReco
         try (FDBRecordContext context = openContext(simpleVersionHook)) {
             FDBRecordVersion version = FDBRecordVersion.incomplete(context.claimLocalVersion());
             FDBStoredRecord<?> stored1 = recordStore.saveRecord(record1, version);
-            assertThat(stored1.hasVersion(), is(true));
-            assertEquals(stored1.getVersion().getLocalVersion(), 0);
-            assertThat(stored1.getVersion().isComplete(), is(false));
+            assertTrue(stored1.hasVersion());
+            assertEquals(0, stored1.getVersion().getLocalVersion());
```

Comment:
Here and a few other places, the test put the expected value as the second argument rather than the first, so the error message from assertion failures would have been backwards. This fixes those to be right. Also not totally related, other than that sonar complained about those problems and I was editing this file.